core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1376,6 +1376,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_JOURNAL_TOLERATE_CORRUPTION =
+      new Builder(Name.MASTER_JOURNAL_TOLERATE_CORRUPTION)
+          .setDefaultValue(false)
+          .setDescription("Whether to tolerate master state corruption."
```

Comment:
Does this control both for replaying journal entries, as well as applying entries during operation? Do we want a single flag to control both scenarios?

Reply:
I changed to only control replaying journal entries (basically `processJournalEntries`) since the reason of this propertykey is for advanced users to ignore some of the journal apply errors when starting master nodes. Thanks